core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/impl/TupleFunctionEvaluationStrategy.java

```diff
@@ -60,6 +64,7 @@ public TupleFunctionEvaluationStrategy(TripleSource tripleSource, Dataset datase
 			long iterationCacheSyncThreshold, EvaluationStatistics evaluationStatistics) {
 		super(tripleSource, dataset, serviceResolver, iterationCacheSyncThreshold, evaluationStatistics);
 		this.tupleFuncRegistry = tupleFuncRegistry;
+		this.setQueryEvaluationMode(QueryEvaluationMode.STANDARD);
```

Comment:
No, tuple function evaluation really should not be part of strict mode (and arguably should not even sit in standard, since it's not a spec-compliant feature). But I didn't want to overcomplicate things that far.